pkg/tests/api/alerting/api_alertmanager_test.go

```diff
@@ -0,0 +1,133 @@
+package alerting
```

Comment:
Alternatively, we could have this test under [pkg/services/ngalert/tests](https://github.com/grafana/grafana/tree/master/pkg/services/ngalert/tests)

Reply:
I originally tried that, but given testinfra is a top-level package it wouldn't let me due to circular dependencies.